### DISCUSSION TITLE
Available options for GraphQL Servlet (graphql.servlet)

### DISCUSSION THREAD

**Discussion by aucampia-emrsn** [graphql-java-kickstart/graphql-spring-boot#543] [Q&A]
What options are available for [GraphQL Servlet (graphql.servlet)](https://github.com/graphql-java-kickstart/graphql-spring-boot#enable-graphql-servlet)? Can  you link to the javadoc of the class that is using these properties?

**Comment by AnandMandhre** [comment]
I saw GraphQLWebAutoConfiguration using these properties

`@EnableConfigurationProperties({GraphQLServletProperties.class})
public class GraphQLWebAutoConfiguration {
......`
